analyses/cell-type-consensus/references/consensus-validation-groups.tsv

```diff
@@ -0,0 +1,53 @@
+consensus_ontology	consensus_annotation	validation_group_ontology	validation_group_annotation
+CL:0000136	adipocyte	CL:0000136	adipocyte
+CL:0000236	B cell	CL:0000236	B cell
+CL:0000945	lymphocyte of B lineage	CL:0000236	B cell
+CL:0000785	mature B cell	CL:0000236	B cell
+CL:0000788	naive B cell	CL:0000236	B cell
+CL:0000787	memory B cell	CL:0000236	B cell
+CL:0000138	chondrocyte	CL:0000138	chondrocyte
+CL:0000451	dendritic cell	CL:0000451	dendritic cell
+CL:0000115	endothelial cell	CL:0000115	endothelial cell
+CL:0000071	blood vessel endothelial cell	CL:0000115	endothelial cell
+CL:2000008	microvascular endothelial cell	CL:0000115	endothelial cell
+CL:0000066	epithelial cell	CL:0000066	epithelial cell
+CL:0002077	ecto-epithelial cell	CL:0000066	epithelial cell
+CL:0000312	keratinocyte	CL:0000066	epithelial cell
+CL:0000076	squamous epithelial cell	CL:0000066	epithelial cell
+CL:0000362	epidermal cell	CL:0000066	epithelial cell
+CL:0000057	fibroblast	CL:0000057	fibroblast
+CL:0008001	hematopoietic precursor cell	CL:0008001	hematopoietic precursor cell
+CL:0000037	hematopoietic stem cell	CL:0008001	hematopoietic precursor cell
+CL:0001065	innate lymphoid cell	CL:0001065	innate lymphoid cell
+CL:0000235	macrophage	CL:0000235	macrophage
+CL:0000148	melanocyte	CL:0000148	melanocyte
+CL:0000650	mesangial cell	CL:0000650	mesangial cell
+CL:0000576	monocyte	CL:0000576	monocyte
+CL:0000192	smooth muscle cell	CL:0000187	muscle cell
+CL:0000187	muscle cell	CL:0000187	muscle cell
+CL:0000188	cell of skeletal muscle	CL:0000187	muscle cell
+CL:0000766	myeloid leukocyte	CL:0000763	myeloid
+CL:0000094	granulocyte	CL:0000763	myeloid
+CL:0000775	neutrophil	CL:0000763	myeloid
+CL:0000113	mononuclear phagocyte	CL:0000763	myeloid
+CL:0000764	erythroid lineage cell	CL:0000763	myeloid
+CL:0000556	megakaryocyte	CL:0000763	myeloid
+CL:0000771	eosinophil	CL:0000763	myeloid
+CL:0000623	natural killer cell	CL:0000623	natural killer cell
+CL:0000540	neuron	CL:0002319	neural cell
+CL:0000127	astrocyte	CL:0002319	neural cell
+CL:0000125	glial cell	CL:0002319	neural cell
+CL:0000126	macroglial cell	CL:0002319	neural cell
+CL:0000095	neuron associated cell	CL:0002319	neural cell
+CL:0000669	pericyte	CL:0000669	pericyte
+CL:0000786	plasma cell	CL:0000786	plasma cell
+CL:0000723	somatic stem cell	CL:0000034	stem cell
+CL:0000034	stem cell	CL:0000034	stem cell
+CL:0000499	stromal cell	CL:0000499	stromal cell
+CL:0000327	extracellular matrix secreting cell	CL:0000499	stromal cell
+CL:0000084	T cell	CL:0000084	T cell
+CL:0000624	"CD4-positive, alpha-beta T cell"	CL:0000084	T cell

```

Comment:
I know these quotes are here because of the comma, but are they likely to cause problems downstream?